assets/js/hooks/utc_date_time_input.js

```diff
@@ -0,0 +1,48 @@
+import { getAttributeOrThrow, parseInteger } from "../lib/attribute";
+
+/**
+ * A hook for client-preprocessed datetime and time input.
+ *
+ */
+const UtcDateTimeInput = {
+  mounted() {
+    this.props = this.getProps();
+    this.el.addEventListener("input", (event) => {
+      event.stopPropagation();
+      event.preventDefault();
+    });
+    this.el.addEventListener("change", (event) => {
+      event.stopPropagation();
+      event.preventDefault();
```

Comment:
These are to avoid the form submission right? In this case we are not really using the form, so we should special-case and render without the form - the same way we do for audio/image inputs, except we don't need a separate components.